app.ts

```diff
@@ -0,0 +1,42 @@
+import { Request,Response } from "express-serve-static-core";
```

Comment:
Not 100% sure what this dependency is, does this work the same?

```suggestion
import express, { Request, Response } from "express";
```